### PR TITLE
remove study deletion notif

### PR DESCRIPTION
**Pull request by AbdelHedhili**
Signed-off-by: Abdelsalem <abdelsalem.hedhili@rte-france.com>

### CODE REVIEW

File: src/main/java/org/gridsuite/study/server/service/NotificationService.java [hunk 1]
```diff
@@ -120,15 +120,6 @@ public void emitStudyChanged(UUID studyUuid, UUID nodeUuid, String updateType, S
                 .build());
     }
 
-    @PostCompletion
```
Comment (archendel) [inline src/main/java/org/gridsuite/study/server/service/NotificationService.java:123]: remove UPDATE_TYPE_STUDY_DELETE constant to avoid future confusion

**Review by archendel — APPROVED** [review]
remove a constant to avoid useless variable

File: src/main/java/org/gridsuite/study/server/service/NotificationService.java (hunk 1 above)
Comment (AbdelHedhili) [reply src/main/java/org/gridsuite/study/server/service/NotificationService.java:123]: done

**Review by FranckLecuyer — APPROVED** [review]
Code review: OK
Tests: OK

**Pull request merged**